### ISSUE TITLE
about dataset and training

### ISSUE THREAD

**Issue by poodarchu** [RenzKa/simlingo#14]
1. if I choose to download the entire dataset from huggingface, do I still need to go through the steps in section data generation?
2. for the training part, which stages in the paper does the training script correspond to? action dreaming or VQA or drive?

**Comment by RenzKa** [comment]
1. The annotated VQA, commentary, and dreamer labels are also included in the Huggingface dataset.

2. The default training is a mixture of all tasks.
